### PR TITLE
add sphinx_rtd_theme

### PR DESCRIPTION
**Pull request by chohner**

### CODE REVIEW

File: recipes/sphinx_rtd_theme/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  imports:
+    - sphinx_rtd_theme
+
+about:
+  home: https://github.com/snide/sphinx_rtd_theme/
+  license: MIT
+  summary: 'ReadTheDocs.org theme for Sphinx, 2013 version.'
+
+  description: |
+    This is a mobile-friendly sphinx theme made for readthedocs.org. It’s currently in
```
Comment (ocefpaf) [inline recipes/sphinx_rtd_theme/meta.yaml]: The `unicode` character in `It’s` is breaking the linter. (We should fix the linter though...)

Comment (chohner) [inline recipes/sphinx_rtd_theme/meta.yaml]: Good to know, thanks.

**Comment by chohner** [comment]
Is there still something tripping up the linter or is it not re-running on the new commit?

File: recipes/sphinx_rtd_theme/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    development there and includes some rtd variable checks that can be ignored if you’re
```
Comment (ocefpaf) [inline recipes/sphinx_rtd_theme/meta.yaml]: The other unicode tripping the linter is here: `you’re`

Comment (chohner) [inline recipes/sphinx_rtd_theme/meta.yaml]: apparently I am blind, thanks :/

Comment (ocefpaf) [inline recipes/sphinx_rtd_theme/meta.yaml]: Oh those are hard to find :smile: 

I am trying to fix the linter so we don't suffer from this.

**Pull request merged**